onyo/lib/filters.py

```diff
@@ -33,30 +39,60 @@ class Filter:
     _arg: str = field(repr=False)
     key: str = field(init=False)
     value: str = field(init=False)
+    operand: str = field(init=False)
```

Comment:
You use "operand" everywhere. But it's the operator / operation. The operands are the arguments to it. So, `value` and `item[self.key]`.

```suggestion
    operator: str = field(init=False)
```

Reply:
Embarrassing. Thanks for the catch!